lib/order.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'menu'
+
+class Order
+
+  attr_reader :ordered_dishes
+
+  def initialize
+    @ordered_dishes = []
+  end
+
+  def add_to_order(dish,quantity)
+    add_to_order  = {dish:dish,quantity:quantity}
+    @ordered_dishes << add_to_order
+  end
+
+  def display_order
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for display_order is too high. [20.64/15]